projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -297,6 +297,7 @@ public static function enqueue_scripts() {
 					'dismissed'  => \Jetpack_Options::get_option( 'dismissed_recommendations', false ),
 				),
 				'isStatsModuleActive'    => $modules->is_active( 'stats' ),
+				'canUserViewStats'       => current_user_can( 'manage_options' ) || current_user_can( 'view_stats' ),
```

Comment:
Presuming we may be adding other types of permissions like this in the project to fix My Jetpack for non-admins, would it be beneficial to add a `permissions` array with `userIsAdmin` and `canUserViewStats` and perhaps other future variables be stored? Just to clean up the structure a bit

Reply:
hmmm good thought. I may save that for a refactor PR since we'd need to rework how we check userIsAdmin from the frontend - we use it in a handful of places.